### PR TITLE
Removed comma from line 317.

### PR DESCRIPTION
**Pull request by tonyxrmdavidson**

### CODE REVIEW

**Comment by tonyxrmdavidson** [comment]
@pb82

File: walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc [hunk 1]
```diff
@@ -314,7 +314,7 @@ NOTE: the pipe symbol (`|`) allows for multiline input, see link:https://yaml.or
 .. Add the following entry to the *volumeMounts* section:
 +
 ----
-- name: backend-config, mountPath: /tmp/keycloak
+- name: backend-config mountPath: /tmp/keycloak
```
Comment (pb82) [inline walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc]: we could also use yaml object syntax: `- {name: "backend-config", mountPath: "/tmp/keycloak" }`

**Comment by wtrocki** [comment]
What is the motivation to use yaml object format?

**Comment by pb82** [comment]
@wtrocki the way that line is rendered in the solution explorer makes it hard to just copy paste it. YAML also allows this object syntax which works better with how its rendered in the browser.

**Comment by wtrocki** [comment]
Awesome.

**Review by pb82 — APPROVED** [review]
Verified, the syntax is correct and accepted by the deployment config

**Pull request merged**